### DISCUSSION TITLE
how to install profiler pack properly?

### DISCUSSION THREAD

**Discussion by YannickDurden** [dunglas/symfony-docker#590] [Q&A] [answered]
Hi,

I installed the profiler pack but when I dump a data within a controller, there's this error:

`Warning: Cannot modify header information - headers already sent by (output started at /app/vendor/symfony/var-dumper/Dumper/AbstractDumper.php:174)`

Also, if I tried to dump withing a twig template, there's this error:

`Error: Allowed memory size of 134217728 bytes exhausted (tried to allocate 98578432 bytes)`

Any idea why it's happening?

Yannick

**Comment by maxhelias** [comment, marked as answer]
Hello,
The first one is a warning not an error. There's already plenty of information on the Internet on this subject.
The second point, you are trying to dump an object that is too large, or your runtime is consuming too much memory for what is allocated to PHP.

**Reply by YannickDurden** [reply]
For the second point, I dumped the global variable "app": {{ dump(app) }}

Without Docker, there's no pb to dump this variable.

**Reply by maxhelias** [reply]
That's what I'm saying, what you're looking to dump is an important object.
Without docker your probably don't have the same configuration and allocated memory for PHP
